packages/ckeditor5-dev-env/lib/release-tools/utils/transformcommitforckeditor5package.js

```diff
@@ -0,0 +1,139 @@
+/**
+ * @license Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+'use strict';
+
+const chalk = require( 'chalk' );
+const { logger } = require( '@ckeditor/ckeditor5-dev-utils' );
+const parserOptions = require( './parser-options' );
+const utils = require( './transform-commit-utils' );
+
+/**
+ * Parses a single commit:
+ *   - displays a log when the commit has invalid format of the message,
+ *   - filters out the commit if it should not be visible in the changelog,
+ *   - makes links to issues and user's profiles on GitHub.
+ *
+ * @param {Commit} commit
+ * @param {Boolean} [displayLog=true] Whether to display the logs.
+ * @returns {Commit}
+ */
+module.exports = function transformCommitForCkeditor5Package( commit, displayLog = true ) {
```

Comment:
> Then, the second question is whether we expect to have more functions related to "CKE 5 packages" and "CKE 5 dev packages"?

Is the question - do we want to use this tool for generating the changelog for the main (`ckeditor5`) package?

I would like to wait for the second "transformCommit" function. In the next PR (which will introduce release tool for this package) I will propose even better the files structure.